docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -54,6 +54,26 @@ For more information about how to write tasks, refer to [Tasks]({{< relref "../t
 
 > **Note:** If a directory contains multiple pages or subdirectories, it is a branch bundle, and it must include an `_index.md` file. A directory containing only one page is a leaf bundle, and the content filename must be `index.md`.
 
-## Useful links
+## Pages and page bundles
 
-- For more information about branch bundles and leaf bundles, refer to [Page bundles](https://gohugo.io/content-management/page-bundles/).
+Each web page generated by Hugo comes from one of three source files:
+- `page/_index.md`: a Hugo branch bundle
+- `page/index.md`: a Hugo leaf bundle
+- `page.md`: a Hugo page
+
+Although each of the preceding examples results in the same URL (`/page/`), Hugo works with each source file differently.
+
+**Branch bundles** (`page/_index.md`) are required to produce page hierarchies.
```

Comment:
```suggestion
**Branch bundles** (`page/_index.md`) are required to produce page hierarchies.  Branch bundle pages are the landing page or mini TOC for the files in a particular folder or directory.
```

Reply:
I think earlier in the file it is discussed that these pages tend to be concept pages though they often can be a mini ToC too.

Reply:
Then maybe "Branch bundle pages can be concept topics,  and may include the mini TOC for the files in a particular folder or directory."